lnbits/core/services.py

```diff
@@ -396,11 +391,51 @@ class PaymentKwargs(TypedDict):
             checking_id="service_fee" + temp_id,
             payment_request=payment_request,
             payment_hash=invoice.payment_hash,
-            pending=False,
+            status=PaymentState.SUCCESS,
         )
     return invoice.payment_hash
 
 
+async def _create_external_payment(
+    temp_id: str,
+    amount_msat: MilliSatoshi,
+    conn: Optional[Connection],
+    **payment_kwargs,
+) -> Payment:
+    fee_reserve_total_msat = fee_reserve_total(amount_msat, internal=False)
+
+    old_payment = await get_standalone_payment(temp_id, conn=conn)
+    if old_payment:
+        if old_payment.pending:
+            raise PaymentError("Payment is still pending.", status="pending")
+        elif old_payment.success:
```

Comment:
nitpick: if `raise` or `return` used on the if branch the else is not required

```suggestion
        if old_payment.success:
```